zlibWrapper/zstd_zlibwrapper.c

```diff
@@ -29,13 +29,14 @@
     - zstd source repository : https://github.com/Cyan4973/zstd
 */
 
-#include <stdio.h>           /* fprintf */
-#include <stdlib.h>          /* malloc */
+#include <stdarg.h>          /* va_list */
```

Comment:
minor point :
I guess `stdarg` is required because of `LOG_WRAPPER`,
which is itself only necessary during debug.

So it would be better to group them together and activated on condition of some specific debug flag.

The goal here is to reduce external dependencies to a minimum, to improve portability perspectives.